src/lib/pages/resolvePageTree.ts

```diff
@@ -161,7 +187,7 @@ export async function resolvePageTree({
     parents,
     targetPage,
     flatChildren: flattenChildren ? flattenTree(targetPage) : undefined
-  };
+  } as any;
```

Comment:
this `any` doesn't seem necessary - tsc compiles fine for me

Reply:
Cleaned up :)